examples/format_seqFISH.py

```diff
@@ -0,0 +1,197 @@
+import functools
+import json
+import os
+from typing import Mapping, Tuple, Union
+
+import click
+import numpy as np
+import pandas as pd
+import skimage.io
+from slicedimage import ImageFormat
+
+from starfish.experiment.builder import FetchedTile, TileFetcher, write_experiment_json
+from starfish.types import Coordinates, Features, Indices, Number
+
+
+# We use this to cache images across tiles.  In the case of the osmFISH data set, volumes are saved
+# together in a single file.  To avoid reopening and decoding the TIFF file, we use a single-element
+# cache that maps between file_path and the npy file.
+@functools.lru_cache(maxsize=1)
+def cached_read_fn(file_path) -> np.ndarray:
+    return skimage.io.imread(file_path)
+
+
+class SeqFISHTile(FetchedTile):
+    def __init__(
+            self,
+            file_path: str,
+            coordinates: Mapping[Union[str, Coordinates], Union[Number, Tuple[Number, Number]]],
+            z: int,
+            ch: int,
+    ):
+        self._file_path = file_path
+        self._z = z
+        self._ch = ch
+        self._coordinates = coordinates
+
+    @property
+    def shape(self) -> Tuple[int, ...]:
+        """Gets image shape directly from the data"""
+        return self.tile_data().shape
+
+    @property
+    def coordinates(self) -> Mapping[Union[str, Coordinates], Union[Number, Tuple[Number, Number]]]:
+        """Stores coordinate information passed from the TileFetcher"""
+        return self._coordinates
+
+    @property
+    def format(self) -> ImageFormat:
+        """Image Format for SeqFISH data is TIFF"""
+        return ImageFormat.TIFF
+
+    def tile_data(self) -> np.ndarray:
+        """vary z the slowest, then channel -- each round has its own TIFF"""
+        return cached_read_fn(self._file_path)[self._z, self._ch]
+
+
+class SeqFISHTileFetcher(TileFetcher):
+
+    def __init__(self, input_dir: str) -> None:
+        """Implement a TileFetcher for a single SeqFISH Field of View."""
+        self.input_dir = input_dir
+
+    @property
+    def coordinates(self) -> Mapping[Union[str, Coordinates], Union[Number, Tuple[Number, Number]]]:
+        """Returns dummy coordinates for this single-FoV TileFetcher"""
+        return {
+            Coordinates.X: (0., 0.),
```

Comment:
LOL.

on a serious note, even when it's a single FOV, it should have a non-zero sizes in physical coordinate space.  i suspect assigning a physical coordinate of a spot will require dividing by the size of the tile in physical coordinate space.